go/vt/worker/split_clone.go

```diff
@@ -789,7 +789,7 @@ func (scw *SplitCloneWorker) waitForTablets(ctx context.Context, shardInfos []*t
 			// We wait for --min_healthy_rdonly_tablets because we will use several
 			// tablets per shard to spread reading the chunks of rows across as many
 			// tablets as possible.
-			if _, err := waitForHealthyRdonlyTablets(ctx, scw.wr, scw.tsc, scw.cell, keyspace, shard, scw.minHealthyRdonlyTablets, timeout); err != nil {
+			if _, err := waitForHealthyTablets(ctx, scw.wr, scw.tsc, scw.cell, keyspace, shard, scw.minHealthyRdonlyTablets, timeout, topodatapb.TabletType_RDONLY); err != nil {
```

Comment:
Hm. Since we're using `RDONLY` tablets for real workloads now, should this check be more specific to the split case?

Reply:
I think it should be ok. We will have to be careful in this case to make sure we don't pass `min` as 1. That we way we will make sure there are enough `RDONLY` tablets that are still serving.